py/desitarget/sv2/sv2_cuts.py

```diff
@@ -313,7 +313,8 @@ def isLRG(gflux=None, rflux=None, zflux=None, w1flux=None, w2flux=None,
     # ADM color-based selection of LRGs.
     lrg &= isLRG_colors(
         gflux=gflux, rflux=rflux, zflux=zflux, w1flux=w1flux,
-        zfiberflux=zfiberflux, south=south, primary=primary
+        zfiberflux=zfiberflux, gaiagmag=gaiagmag, south=south,
```

Comment:
The `gaiagmag=gaiagmag` here, needs to instead be in the call to `notinLRG_mask()` (about 10 lines further up), as that is the function in which you actually apply the limits on Gaia G.